### PR TITLE
Details& Comments page

### PR DESCRIPTION
**Pull request by DorisW3**
- added a new entry form to create a comment
- the comment can be sent by clicking the create button, the comment will be displayed on top of the older entries
- displayed an array of "simulated, older entries", from newest to oldest from the top  
- added a little styling

### CODE REVIEW

File: components/EntriesList.js [hunk 1]
```diff
@@ -1,9 +1,6 @@
-import "./EntriesList.css";
-import { FavoriteButton } from "../FavoriteButton/FavoriteButton";
-
 export function EntriesList({ date, title, comment }) {
   return (
-    <div className="entries__section-entry">
+    <section className="entries__section-entry">
       <time className="entry-date">{date}</time>
```
Comment (alperkuertuel) [inline components/EntriesList.js]: I guess we should use styled components for styling. Thats what the coaches said. Do you plan to implement it later?

File: components/PictureDetails.js [hunk 2]
```diff
@@ -1,15 +1,34 @@
...
 import { styled } from "styled-components";
+import { useRouter } from "next/router";
 
 export default function PictureDetails() {
+  const router = useRouter();
+  const { id } = router.query;
+
+  const selectedPicture = pictures.find(
+    ({ slug }) => slug === router.query.slug
+  );
```
Comment (alperkuertuel) [inline components/PictureDetails.js]: I guess to create the slug functionality you should create another story not add it in this story as a side task.

File: components/EntriesList.js (hunk 1 above)
Comment (DorisW3) [reply components/EntriesList.js]: I thought at first I would need it. But as you can see in the code I did the styling with styled components and forgot to delete these classNames ;-)

File: components/Overview.js [hunk 3]
```diff
@@ -1,22 +1,23 @@
...
-          />
-
+        <StyledListItem key={picture.id}>
+          <Link href={`/detailscomments/${picture.id}`}>
+            <StyledImage
+              src={picture.image}
+              alt={picture.theme}
+              height={300}
+              width={200}
+              priority={true} // bei Bildern mit großer Datenmenge, verbessert der Code das laden
```
Comment (jonas8900) [inline components/Overview.js:18]: wow very good !

File: components/PictureDetails.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        priority={true} // bei Bildern mit großer Datenmenge, verbessert der Code das laden
+      />
+      <p>{selectedPicture.theme}</p>
+      <p>{selectedPicture.description}</p>
+      <p>{selectedPicture.username}</p>
+    </StyledArcticle>
+  );
+}
+
+const StyledArcticle = styled.div`
```
Comment (jonas8900) [inline components/PictureDetails.js:32]: maybe you could name the div not like Article

File: components/PictureDetails.js (hunk 2 above)
Comment (jonas8900) [reply components/PictureDetails.js]: good point, seems it were changed

**Comment by jonas8900** [comment]
Looks very good and structured to me !

**Comment by jonas8900** [comment]
you should fix that when you add a comment, you can see the comment in all of the picturepreviews.

**Comment by DorisW3** [comment]
Thanks, for the QA! I have planned this for a later user stories :-)

**Pull request merged**